Hush/Assets/Scripts/Game/GameMaster.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameMaster : MonoBehaviour
+{
+    public static GameMaster gm;
```

Comment:
Don't really need this you can just have a single GO with the GameMaster component and then you can call "GetComponentOfType<GameMaster>()" anywhere and get the game master!

Reply:
Although I guess that's fine! I like that too!